imports/plugins/core/checkout/client/components/completedShopOrders.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
```

Comment:
I intended to extend this component and its children but they didn't write these components in an extendable way. They used relative imports of the components instead of relying on Reaction  Commerce's `Component` object.

Reply:
Ok, but you will always need to edit this file for every upgrade.